--- libs/basekit/source/Common_inline.h.orig	2011-09-06 15:44:55.000000000 +0900
+++ libs/basekit/source/Common_inline.h	2011-09-06 16:01:03.000000000 +0900
@@ -53,14 +53,14 @@
 #if defined(__APPLE__) 
 
-	#define NS_INLINE static __inline__ __attribute__((always_inline))
+	#define IO_NS_INLINE static inline
 
 	#ifdef IO_IN_C_FILE
 		// in .c 
 		#define IO_DECLARE_INLINES
-		#define IOINLINE NS_INLINE
+		#define IOINLINE IO_NS_INLINE
 	#else
 		// in .h 
 		#define IO_DECLARE_INLINES
-		#define IOINLINE NS_INLINE
+		#define IOINLINE IO_NS_INLINE
 	#endif 	
 
